src/js/components/Stars.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+export default class Stars extends React.Component{
+
+
+
+    render(){
+
+        var fillStarTemp = [],
```

Comment:
to możemy napisać dużo prościej:
- ustalmy, że w propsach przychodzą dwie dane: maksymalna ilość gwiazdek, zaznaczona ilość gwiazdek
- przed render tworzysz sobie pętle od 1 do ilości gwiazdek i tam sprawdzasz czy i jest <= zaznaczona ilość
- w render wykorzystujesz zbudowaną ocenę